packages/react-celo/src/components/button.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+
+import cls from '../utils/tailwind';
+import useTheme from '../utils/useTheme';
+
+const styles = cls({
+  button: `
+    tw-rounded
+    tw-px-2
+    tw-py-1
+    tw-font-medium`,
+});
+
+type anchorType = JSX.IntrinsicElements['a'];
+type buttonType = JSX.IntrinsicElements['button'];
+type extendType = anchorType & buttonType;
+
+interface Props extends extendType {
+  as?: 'a' | 'button' | React.ComponentType<unknown>;
+  className?: string;
+  style?: Record<string, string | number>;
+}
+
+export default function Button({
+  as = 'button',
```

Comment:
id like to make this case that this should be required so we dont accidentally make buttons when anchor tags are the semantically correct thing to do